signxml/__init__.py

```diff
@@ -370,9 +370,9 @@ def sign(
             self.id_attributes = (id_attribute,)
 
         if isinstance(cert, (str, bytes)):
-            cert_chain = list(iterate_pem(cert))
+            self.cert_chain = list(iterate_pem(cert))
```

Comment:
Please do not attach any signature-specific information to the signer/verifier instance. The instance is stateless.